controllers/datamover.go

```diff
@@ -20,6 +22,19 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
+const (
+	ResticPassword   = "RESTIC_PASSWORD"
+	ResticRepository = "RESTIC_REPOSITORY"
+	ResticsecretName = "dm-restic-secret"
```

Comment:
this will be changed to 'dm-credential' right?

Reply:
yes. Submitting a commit right away

Reply:
done